package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "server": "node -r esm server/server.js",
     "server-nodemon": "nodemon -r esm server/server.js",
-    "build": "del build && npm run build-directories && npm run build-images && npm run build-js && npm run build-css && npm run prefix-css && npm run move-images && npm run move-fonts && npm run revision-js && npm run revision-css && npm run optimize-svgs",
+    "build": "del build && npm run build-directories && npm run build-js && npm run build-css && npm run prefix-css && npm run move-images && npm run move-fonts && npm run revision-js && npm run revision-css && npm run optimize-svgs",
```

Comment:
What happened with the npm run build-images script?

Reply:
Removed because it was unnecessary. Images are still included in the build though.